event-message/tests/case-01.http

```diff
@@ -0,0 +1,36 @@
+POST http://localhost:8080/api/v1/incidents
+
+{
+  "number": "+94749933244",
+  "incident": {
+      "description": "System breakdown",
+      "date": "2023-10-10",
+      "time": "10:00"
+    }
+}
+
+###
+
+POST http://localhost:8080/api/v1/incidents
```

Comment:
need to go to a new file or we can remove these since it's sort of a duplication.

Reply:
Included multiple different requests to imitate multiple events.